sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -422,7 +431,18 @@ def delete_configuration_setting(self, key, label=None, **kwargs):
                 key="MyKey", label="MyLabel"
             )
         """
+        key = kwargs.pop("key", None)
```

Comment:
Can user uses keyword arg to set ConfigSetting?

And what is the behavior of delete_config_setting("xxx", key=yyy")?

Reply:
Updated this.
The behavior would be to delete "xxx"

Reply:
Similar question:
what's the behavior of delete_config_setting(config_setting=ConfigSetting, key="xxx") & delete_config_setting(ConfigSetting (positional), key="xxx")?

Reply:
For the first, it would be "xxx" (`config_setting` is not a supported kwarg)
For the second, ConfigSetting.key would be the value used

Reply:
You mean 
delete_config_setting(config_setting=ConfigSetting) 
does not work?

Reply:
no